patches/lua.CMakeLists.txt

```diff
@@ -0,0 +1,146 @@
+# Lua can be compiled as either C or C++.
```

Comment:
What's the license of this file? Any copyright notice we need to keep?

Reply:
MIT license https://github.com/microsoft/vcpkg/blob/master/ports/lua/CMakeLists.txt

Reply:
Then we need to comply with:

> The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.

Reply:
Done

Reply:
Thanks!